.babelrc

```diff
@@ -1,16 +1,64 @@
 {
-    "plugins": [
-        [
-        "babel-plugin-transform-builtin-extend",
-        {
-            "globals": [
-            "Error"
-            ]
-        }
+  "plugins": [
```

Comment:
I think it would be helpful to alphabetize this `plugins` array. I would alphabetize based on the part after `babel-plugin` or `@babel/plugin`.

Reply:
I copied this plugins array from babel documentations, they dropped support for presets, now it should be a list of plugins. Plugins order matters, I`d prefer not to change it and leave as it is written in docs
https://www.npmjs.com/package/@babel/preset-stage-0

Reply:
Regarding [this](https://www.npmjs.com/package/@babel/preset-stage-0) we need only `@babel/plugin-proposal-function-bind` for stage-0. Why did you add all suggested plugins from stage 1, 2, 3?
If I understand right we need only one to make it work, like it was. And all other will be like redundant dependencies

Reply:
We aren't using the proposed function bind syntax (it uses the `::` operator). So, we don't need stage-0. I am guessing we don't need any any of these `plugin-proposal-*` plugins. Can we remove everything we don't need?